app/models/analysis.js

```diff
@@ -7,8 +7,12 @@ var mongoose = require("mongoose"),
   Msa = require(__dirname + "/msa"),
   extend = require("mongoose-schema-extend");
 
+var redis = require('redis'),
+  client = redis.createClient({host : 'localhost', port : 6379});
```

Comment:
We might want to make the redis hort and port configurable in one location - like setup.js